hips/draw/simple.py

```diff
@@ -117,6 +117,38 @@ def _fetch_tiles(self) -> HipsTile:
             tile = HipsTile.fetch(tile_meta, url)
             yield tile
 
+    @staticmethod
+    def _measure_tile_shape(corners: tuple) -> List[list]:
+        """Compute tile edges and diagonals."""
+
+        def compute_distance(i: list, j: list) -> float:
```

Comment:
I find this way you've implemented `compute_distance` hard to read.
My suggestion would be that you move the
```
x, y = corners
```
to the top and then do this:
```
def compute_distance(i, j):
    return np.sqrt( (x[i] - x[j]) ** 2 + (y[i] - y[j]) ** 2)
```
i.e. have i and j as the integer index of the corner.

Then you should be able to call it like this:
```
edge_lengths = [compute_distance(i, (i+1) %4) for i in range(4)]
```
and like this
```
diagonal_lengths = [compute_distance(0, 2), compute_distance(1, 3)]
```
Note that I didn't try this code, I just typed it here.
Can you please try to change the implementation to work this way?